docs/general/clients/kodi.md

```diff
@@ -172,13 +172,13 @@ JellyCon behaves more like a standard Kodi streaming add-on. Media is accessed p
 
 ### Configuring Home
 
-Many Kodi skins allow for customizing of the home menu with custom nodes and widgets. However, all of these use slightly different layouts and terminology. Rather than a step-by-step guide, this section serves as an barebones introduction to customizing a skin.
+Many Kodi skins allow for customizing of the home menu with custom nodes and widgets. However, all of these uses slightly different layouts and terminology. Rather than a step-by-step guide, this section serves as a bare-bones introduction to customizing a skin.
```

Comment:
```suggestion
Many Kodi skins allow for customizing of the home menu with custom nodes and widgets. However, all of these use slightly different layouts and terminology. Rather than a step-by-step guide, this section serves as a bare-bones introduction to customizing a skin.
```